src/main/java/com/sap/cloudfoundry/client/facade/CloudControllerClient.java

```diff
@@ -385,7 +386,7 @@ void createApplication(String applicationName, Staging staging, Integer disk, In
      * @param applicationName the name of the application
      * @return the list of recent log entries
      */
-    List<ApplicationLog> getRecentLogs(String applicationName);
+    List<ApplicationLog> getRecentLogs(String applicationName, OffsetDateTime offset);
```

Comment:
Is there a real benefit to using OffsetDateTime? Why not the simpler LocalDataTime